tests/datavolume_test.go

```diff
@@ -2579,17 +2581,29 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 			var err error
 			By(fmt.Sprintf("creating storage class %s", scName))
 			sc := utils.DefaultStorageClass.DeepCopy()
-			sc.Name = scName
-			sc.ResourceVersion = ""
 			sc.Annotations[controller.AnnDefaultStorageClass] = "false"
+			sc.Annotations[controller.AnnDefaultVirtStorageClass] = "false"
+			sc.ObjectMeta = metav1.ObjectMeta{
+				Name: scName,
+				Labels: map[string]string{
+					"cdi.kubevirt.io/testing": "",
+				},
+				ResourceVersion: "",
+				Annotations:     sc.Annotations,
+			}
```

Comment:
you may be interested in CreateNonDefaultVariationOfStorageClass

Reply:
I mean we don't supply a virt-default sc to the tests. You can physically do it though, just going to result in lots of failures